fbw-a380x/src/systems/systems-host/systems/LegacyGpws.ts

```diff
@@ -4,888 +4,935 @@ import { LegacySoundManager, soundList } from 'systems-host/systems/LegacySoundM
 import { A380X_DEFAULT_RADIO_AUTO_CALL_OUTS, A380XRadioAutoCallOutFlags } from '../../shared/src/AutoCallOuts';
```

Comment:
Please split the format changes into a separate commit. It's impossible to review with them mixed in.

Reply:
That's why https://github.com/flybywiresim/aircraft/pull/9013 exists and has to be merged before this one

Reply:
But the changes seem to be mixed in the same commit here?

Reply:
Just had a look again, I didn't change anything in LegacyGpws, don't know why I linted it in the first place then :/ I reverted the change on this file.